packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -717,5 +718,182 @@ describe('build-schema', () => {
         expect(schema).to.deepEqual(expectedSchemaForCategory);
       });
     });
+
+    it('converts HasMany and BelongsTo relation links', () => {
+      @model()
+      class Product extends Entity {
+        @property({id: true})
+        id: number;
+
+        @belongsTo(() => Category)
+        categoryId: number;
+      }
+
+      @model()
+      class Category extends Entity {
+        @property({id: true})
+        id: number;
+
+        @hasMany(() => Product)
+        products?: Product[];
+      }
+
+      @model()
+      class CategoryWithoutProp extends Entity {
+        @hasMany(() => Product)
+        products?: Product[];
+      }
+      const expectedSchema: JsonSchema = {
+        definitions: {
+          ProductWithRelations: {
+            title: 'ProductWithRelations',
+            properties: {
+              id: {type: 'number'},
+              categoryId: {type: 'number'},
+              category: {$ref: '#/definitions/CategoryWithRelations'},
+            },
+          },
+        },
+        properties: {
+          id: {type: 'number'},
+          products: {
+            type: 'array',
+            items: {$ref: '#/definitions/ProductWithRelations'},
+          },
+        },
+        title: 'CategoryWithRelations',
+      };
+      const jsonSchema = getJsonSchema(Category, {includeRelations: true});
+      expect(jsonSchema).to.deepEqual(expectedSchema);
+
+      // To check for case when there are no other properties than relational
```

Comment:
Please move this check to a new test case. 

It's a best practice to keep one (logical) assertion per test.

Reply:
fixed.